.env.example

```
@@ -0,0 +1,14 @@
+# Required
+REACT_APP_TINK_LINK_PERMANENT_USERS_CLIENT_ID=
+TINK_LINK_PERMANENT_USERS_CLIENT_SECRET=
```

```suggestion
REACT_APP_TINK_CLIENT_ID=
TINK_CLIENT_SECRET=
```
They are not really tied to a single use case, I see why name them by the context here–but unsure if that's needed?